dev/bench/bench.sh

```diff
@@ -72,14 +72,16 @@ check_variable () {
 : "${old_coq_opam_archive_git_uri:=https://github.com/coq/opam-coq-archive.git}"
 : "${new_coq_opam_archive_git_branch:=master}"
 : "${old_coq_opam_archive_git_branch:=master}"
+: "${new_coq_version:=8.19+rc1}"
```

Comment:
```suggestion
: "${new_coq_version:=dev}"
```
maybe this will work better, lots of packages don't have a 8.19 published yet